tests/DefaultsSchemaValidatorTest.php

```diff
@@ -21,23 +21,23 @@ final class DefaultsSchemaValidatorTest extends BaseCodegenTestCase {
   public function testBoolean(): void {
```

Comment:
This file was completely unclear if it cares about array types, defaulting to darrays.